### PR TITLE
Add make release script and notes to README

### PR DESCRIPTION
**Pull request by MattMS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,11 @@ For more details on configuring the server, see [http-server (0.8.x)][http-serve
 
 [http-server]: https://github.com/indexzero/http-server/tree/0.8.5#available-options
 
+
+## What to do on a new release
+
+1. Update [package.json](./package.json) to latest version of `ramda`.
+
+2. Install packages: `npm i`
```
Comment (buzzdecafe) [inline README.md]: why not roll this into the `make_release` step?

File: README.md [hunk 2]
```diff
@@ -21,3 +21,11 @@ For more details on configuring the server, see [http-server (0.8.x)][http-serve
...
+
+3. `npm run make_release`
```
Comment (buzzdecafe) [inline README.md]: i'd like to see all three of these rolled into one, so i can invoke `npm run make_release $VERSION` and have it update package.json, install packages and build the docs. 

However, we may want a way to rebuild docs for and old version without updating package.json as well.

**Comment by MattMS** [comment]
The `copy_to_version_folder.js` script also copies `package.json`, which has the current version of everything, so you can run all the scripts for that version inside the folder.

As far as running in a single step goes, the first 2 steps could maybe be completed with `npm i ramda@x.y.z --save-dev`, but I'm not sure how to compress it all in cross-platform way without another script.

**Comment by buzzdecafe** [comment]
merging as is for now -- may circle back later. thanks

**Pull request merged**